rules/codelingo/effective-go/comment-first-word-when-empty/codelingo.yaml

```diff
@@ -21,6 +21,22 @@ funcs:
 tenets:
   - name: comment-first-word-when-empty
     actions:
+ codelingo/docs:
+        title: Comment First Word as Subject
+        body: |
+          Each exported function should have a doc comment with a summary of the function
+          to allow for an automated presentation of these doc comments.
+          The first word in the summary should be the function name, followed by a description. 
+          
+          ## Bad code
+          ```
+          func Unnamed(){}
+          ```
+          ## Good code
+          ```
+          //Unnmaded is the first word and this is a summary of the function. 
+          func Unnamed(){}
+          ```
       codelingo/review:
         comment: |
           Every exported function in a program should have a doc comment. The first sentence should be a summary that starts with the name ({{funcName}}) being declared.
```

Comment:
Perhaps wrap ({{funcName}}) in backticks to get markdown code formatting? i.e.

`({{funcName}})`